quickstart/gcp-cloudshell-quickstart.py

```diff
@@ -119,18 +217,18 @@ def quickstart_config_read(cfg_file):
 
 def ad_password_get():
     txt = r'''
-    Please enter a password for the Active Directory Administrator.
-
-    Note Windows password complexity requirements:
-    1. Must not contain user's account name or display name
-    2. Must have 3 of the following categories:
-       - A-Z
-       - a-z
-       - 0-9
-       - special characters: (~!@#$%^&*_-+=`|\(){}[]:;"'<>,.?/)
-       - unicode characters
-
-    See: https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements
+Please enter a password for the Active Directory Administrator.
```

Comment:
use this instead? https://docs.python.org/3/library/textwrap.html#textwrap.dedent